common/src/main/java/com/wynntils/screens/settings/WynntilsBookSettingsScreen.java

```diff
@@ -54,9 +55,7 @@ public final class WynntilsBookSettingsScreen extends WynntilsScreen implements
     private ScrollButton configurableListScrollButton;
     private ScrollButton configListScrollButton;
 
-    // FIXME: Suboptimal to have both selected variables, but we need big changes to fix that
-    private Feature selectedFeature = null;
-    private Overlay selectedOverlay = null;
+    private AbstractConfigurable selected = null;
```

Comment:
Overall this looks good, but you should use the `Configurable` interface to define the type everywhere (except when it is actually instantiated).